CDCgov/prime-reportstream
# The elims transform just negated the default transform and since none…

… of the things in the default transform actually belong there, I renamed it to something more appropriate, deleted the elims sender transform, and updated the places that it needed to be updated in the code. Once this gets pushed out staging will need to be updated so that they no longer have a sender transform.

This PR renames the default transform to something more appropriate, removes it from use for ELIMS, deletes the ELIMS transform since that is only used to negate the default one.

Test Steps:
1. Verify that the ELIMS sender is not using a sender transform anymore.
2. Make sure the tests still pass

## Changes
- Renames the default transform to something more appropriate
- Removes it from use for ELIMS
- Deletes the ELIMS transform since that is only used to negate the default one.


## Checklist

### Testing
- [ X] Tested locally?
